packages/scripts/preset/config/webpack.config.js

```diff
@@ -61,7 +61,7 @@ module.exports = ({ getUserConfig }) => {
 	return {
 		entry: ['babel-polyfill', 'whatwg-fetch', `${process.cwd()}/src/app/index.js`],
 		output: {
-			path: `${process.cwd()}/dist`,
+			path: `${process.cwd()}/build`,
```

Comment:
why build ?
I don't really like it, build folder make me feel like `tmp`
dist stands for distribution and is the default value in webpack

Reply:
👍 Thanks for the info, I just removed it to use webpack default value and adapt devServer to server static from this default folder